### DISCUSSION TITLE
How about requesting more resource to our dev namespace?

### DISCUSSION THREAD

**Discussion by thermcampos** [bcgov/quickstart-openshift#925] [General]
Hey guys! I'm wondering about requesting more resources to our OpenShift dev namespace. (Not sure if it can be done only to a namespace or must be done to all once, since it's the same project code).

- First question: How usual is to request more resources?
- Second: Since we have three services running at the same project (and more in the namespaces, due to PR deployments) I think would be justifiable that request. Do you guys agree?

Note: Although we should be releasing native images with Spring Boot soon, I still think we may run out of quota every now and then.

Should I mention people here?

cc: @DerekRoberts @mishraomp @webgismd @annibalsilva @paulushcgcj

**Comment by mishraomp** [comment]
@RMCampos 
here is the link to the documentation around openshift and resource quotas et..
https://docs.developer.gov.bc.ca/openshift-project-resource-quotas/

> If the default allocations aren't sufficient for your application, [you can ask for a quota increase](https://docs.developer.gov.bc.ca/request-quota-increase-for-openshift-project-set/). You'll need a Sysdig dashboard that shows that your application needs more of a specific resource type (CPU, RAM or storage) in a specific namespace. Provide this proof to the Platform Services team before they can approve a quota increase that you submit in the [Platform Project Registry](https://registry.developer.gov.bc.ca/public-landing).

cc @DerekRoberts @webgismd @annibalsilva @paulushcgcj

**Reply by paulushcgcj** [reply]
JVM apps tend to be a resource hunger, so increasing the resources for some time may be an option or maybe tinkering with the base image to use one that can help with that as well. 

Another solution would be to try to move to native, but I've been having some issues with slf4j and GraalVM.

**Comment by DerekRoberts** [comment]
@RMCampos Are you able to request resources here?  If not, we should figure out who the Product Owner or Technical Lead is.
https://registry.developer.gov.bc.ca/dashboard

**Reply by thermcampos** [reply]
I think so. Sabina is the PO, and the TL is me. I could log in, but there's no product. Should I create one for SPAR?

**Reply by DerekRoberts** [reply]
There should be an existing product.  See if Ziad or Sabina have access?  (Michelle and I don't.)

**Reply by SLDonnelly** [reply]
Hello! So I can't see a product in the dashboard yet. I think that Ziad will need to transfer it over to me. I'll review the documentation that Om shared regarding openshift and resource quotas. I've got a meeting with Ziad to go over all this since this is all new to me...I'll update the ticket when request is submitted

@RMCampos @DerekRoberts
